besu/src/test/resources/everything_config.toml

```diff
@@ -123,10 +123,12 @@ rpc-ws-authentication-jwt-public-key-file="none"
 rpc-ws-ssl-enabled=false
 rpc-ws-ssl-keystore-file="none.pfx"
 rpc-ws-ssl-keystore-password="none.passwd"
+rpc-ws-ssl-keystore-password-file="none.txt"
 rpc-ws-ssl-keystore-type="none"
 rpc-ws-ssl-client-auth-enabled=false
 rpc-ws-ssl-truststore-file="none.pfx"
 rpc-ws-ssl-truststore-password="none.passwd"
+rpc-ws-ssl-truststore-password-file="none.txt"
```

Comment:
does it make sense to specify both password and password-file or should we only allow one or the other?

Reply:
that should be handled by the ArgGroup